tests/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixerTest.php

```diff
@@ -191,4 +201,29 @@ public function provideFix81Cases(): iterable
             '<?php class T { public function Foo(object $A): NEVER {die;}}',
         ];
     }
+
+    /**
+     * @dataProvider provideFix82Cases
+     *
+     * @requires PHP 8.2
+     */
+    public function testFix82(string $expected, string $input): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFix82Cases(): iterable
+    {
+        foreach (['true', 'false', 'null'] as $type) {
+            yield sprintf('standalone type `%s`', $type) => [
+                sprintf('<?php class T { public function Foo(%s $A): %1$s {return %1$s;}}', $type),
+                sprintf('<?php class T { public function Foo(%s $A): %1$s {return %1$s;}}', strtoupper($type)),
```

Comment:
IMHO:
- it's not readable enough, would be better with explicit `null`
- `return null;` is not required here because `null` here is not a _type_, it's a scalar _value_. And that's why tests are failing because `NativeFunctionTypeDeclarationCasingFixer` is related to types so it's not its responsibility to fix values' casing.

Reply:
thanks updated :+1:

Reply:
if you want, feel free to take over this PR, I'm juggling to many PR's already :sweat_smile:  and I think you know more about this change in PHP8.2

Reply:
Ok, I can do it, but how to do it technically? Should I take this branch to my fork and open new MR?

Reply:
Yeah, or on your fork make a new branch from latest, than cherrypick the one commit from here (I already squashed the commits), than make a new PR and we should be good :)

Reply:
@SpacePossum OK, I've created #6623 🙂